File: src/gasing/alessandro_rumampuk/division/backup/division8.py
import time

a = 123456789
b = 8

# GASING Division
start = time.time()
for _ in range(10000000):
        hasil = a // 4 // 2
end = time.time()
gasing_time = end - start
print("GASING Division 8:", gasing_time, "seconds")
<<<<<<< HEAD
=======
print("GASING Result:", hasil)
>>>>>>> f04baed (add and edit)

# Computer Division
start = time.time()
for _ in range(10000000):
    hasil = a // b
end = time.time()
computer_time = end - start
print("General Division 8:", computer_time, "seconds")
<<<<<<< HEAD
=======
print("Computer Result:", hasil)
>>>>>>> f04baed (add and edit)

print("\nGASING is", computer_time/gasing_time, "times faster")

File: src/gasing/alessandro_rumampuk/division/backup/division8.rs
use std::time::Instant;

fn main() {
    let a: i64 = 123456789;
    let b: i64 = 8;

    // GASING Division
    let start = Instant::now();
    let mut hasil: i64 = 0;
    for _ in 0..10000000 {
        hasil = a / 4 / 2;
    }
    let gasing_time = start.elapsed().as_secs_f64();
    println!("GASING Division 8: {} seconds", gasing_time);
<<<<<<< HEAD
=======
    println!("GASING Result: {}", hasil);
>>>>>>> f04baed (add and edit)

    // Computer Division
    let start = Instant::now();
    for _ in 0..10000000 {
        hasil = a / b;
    }
    let computer_time = start.elapsed().as_secs_f64();
    println!("General Division 8: {} seconds", computer_time);
<<<<<<< HEAD
=======
    println!("Computer Result: {}", hasil);
>>>>>>> f04baed (add and edit)

    println!("\nGASING is {} times faster", computer_time/gasing_time);
}